setup.py

```diff
@@ -33,11 +33,15 @@
         "marshmallow>=3.0.0",
         "SQLAlchemy>=1.0.0",
         "Werkzeug>=0.11",
+        "konch>=4.0",
```

Comment:
this will get installed into prod too, I guess it's ok. would be possible to make it an optional dep but possibly not worth it? or maybe it's not even common practice, you'd know better

Reply:
I think it's actually useful to have in prod. When diagnosing issues, I often SSH, install ipython, then import the models.

I considered making this an optional dep, but extras are kinda a pain to deal with in our pip-tools setup. konch is a small (one module) dep without any transitive deps itself.